### PR TITLE
Updated readable-stream@3

### PR DESCRIPTION
**Pull request by mcollina**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 'use strict'
 
-var PassThrough = require('readable-stream').PassThrough
-var inherits = require('inherits')
-var p = require('process-nextick-args')
+const { PassThrough } = require('readable-stream')
```
Comment (vweevers) [inline index.js]: Use of `const` might mean loss of browser support (not that there was any official support).

Comment (mcollina) [reply index.js]: There wasn't any. If there is, I'm happy to revisit.

**Comment by mcollina** [comment]
Thanks for checking @vweevers!

File: index.js (hunk 1 above)
Comment (mcollina) [reply index.js]: Are you using this in the browser?

**Comment by vweevers** [comment]
Is this still needed?

https://github.com/mcollina/cloneable-readable/blob/f9c22547915097cbd27097b15e75130b47adec6a/index.js#L64-L72

File: index.js [hunk 2]
```diff
@@ -65,10 +65,9 @@ Cloneable.prototype._destroy = function (err, cb) {
   if (!err) {
     this.push(null)
     this.end()
-    this.emit('close')
```
Comment (mcollina) [inline index.js]: This line is not needed anymore @vweevers! As for the rest unfortunately yes.

Comment (mcollina) [reply index.js]: No. This library cheats some of the internals to make it work.

File: index.js (hunk 1 above)
Comment (vweevers) [reply index.js]: No, I'm not. Was just an observation :) It's fine, esp. now that it's a conscious decision

**Pull request merged**